### DISCUSSION TITLE
Slack

### DISCUSSION THREAD

**Discussion by jimmyhoran** [MobileNativeFoundation/discussions#28] [General]
I know it's earl days, but just wanted to leave this thought here. It would be great to see either a new Slack channel created on one of the existing Slack groups or a new Slack group created dedicated to this foundation.

**Comment by keith** [comment]
Right now we're hoping to keep most discussion on GitHub discussions because we want them to be publicly searchable, and a bit more weighted towards long form vs synchronous + immediate.

**Reply by jimmyhoran** [reply]
Great to know. Thanks @Keith 🙏

**Reply by piannaf** [reply]
Hey @keith, this thread is a few months old, and I noticed a link to https://mobilenativefdn.slack.com/ in the [Bluepill announcement](https://github.com/MobileNativeFoundation/discussions/discussions/112)

Is there a policy around invites to that workspace? And what's the current thinking around where discussion should happen?

**Reply by keith** [reply]
The current recommendation is still to have discussions on github whenever possible, there is that slack group but at this point it's mostly a ghost town since I'm still encouraging everyone to talk here for the same reasons posted above.
